src/help/components/results/Results.tsx

```diff
@@ -99,29 +100,64 @@ const Results = ({
   );
 
   useEffect(() => {
-    replaceLocation(searchValue);
+    replaceLocation(searchValue, isReleaseNotes);
 
     return replaceLocation.cancel;
-  }, [replaceLocation, searchValue]);
+  }, [replaceLocation, searchValue, isReleaseNotes]);
+
+  const searchNode = (
+    <div className={styles['results-header']}>
+      <img
+        src={HelperImage}
+        className={styles.helper}
+        width="290"
+        height="123"
+        alt=""
+      />
+      <strong className={cn('tiny', styles.title)}>
+        <HelpIcon width="0.7em" height="0.7em" />
+        {isReleaseNotes ? 'Release notes ' : 'Help '}search results
+      </strong>
+      <SearchInput
+        isLoading={initialLoading}
+        onChange={(event: ChangeEvent<HTMLInputElement>) =>
+          setSearchValue(event.target.value)
+        }
+        placeholder="Search"
+        value={searchValue}
+        autoFocus
+      />
+    </div>
+  );
 
   let main: ReactNode = null;
 
-  if (dataObject.loading && !dataObject.data) {
-    main = <Loader progress={dataObject.progress} />;
-  } else if (dataObject.error || !dataObject.data) {
-    main = <ErrorHandler status={dataObject.status} />;
-  } else if (!dataObject.data.results.length) {
-    main = <NoResultsPage />;
+  if (initialLoading) {
+    main = <Loader progress={progress} />;
   } else {
     main = (
-      <DataList
+      <DataListWithLoader
         getIdKey={getIdKey}
-        data={dataObject.data.results}
+        data={allResults}
```

Comment:
The type 'HelpAPIModel' has been added to 'APIModel'. But it is still complaining